### ISSUE TITLE
Couldn't find icon with name 'ic_check_box'

### ISSUE THREAD

**Issue by thresholdpeople** [scztt/IconSet.quark#1]
Hello,

I just installed your quark, ran Material.fetch, and upon recompiling received the following error:

ERROR: Couldn't find icon with name 'ic_check_box' in paths: `all the folders within ..../downloaded-quarks/material-design-icons`
followed by the CALL STACK: Exception printing in the post window. 

Seems like maybe the files or file names for the icon repository have been changed...? 

Many thanks for your time and work!
Boris

**Comment by scztt** [comment]
Hmm - the icon `ic_check_box` is not used anywhere in this Quark. Do you have any idea which code might be referring to this icon name? The "Couldn't find icon..." error only occurs when you try to create an icon that doesn't exist - someone, somewhere, must be calling `Material(\ic_check_box)`....

**Comment by scztt** [comment]
I see that Material recently moved to a new major version that completely changes the folder structure. It will take some substantial changes to make this work with `IconSet` - for now, you can go to the folder where the `Material` icons are checked out (this should be `~/Library/Application Support/SuperCollider/downloaded-quarks/material-design-icons` on mac), and run `git checkout 3.0.2`. This will revert to the older, supported version of the Material icons.

**Comment by thresholdpeople** [comment]
Yep, the error was caused on recompiling the class library, I hadn't even gotten a chance to try anything yet. Switching to version 3.0.2 fixed it though, thanks!
